kafka/config_sasl_aws.go

```diff
@@ -0,0 +1,7 @@
+package kafka
+
+type AWS struct {
```

Comment:
@AdrienneCohea Could you please rename the struct to `AWSConfig` to keep it consistent with the existing SASL Mechanisms implementation?